src/components/features/Aircraft.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import aircraftData from "../initialData/aircraftData";
+
+function Aircraft({ selectedAircraft }) {
+  function onSelect(event) {
+    selectedAircraft(event.target.value);
+  }
```

Comment:
I think you can remove this function as well and call the function from the props directly where you use onSelect